localization/i18n/OrcaSlicer.pot

```diff
@@ -9398,7 +9398,7 @@ msgstr ""
 msgid "Flushing volumes for filament change"
 msgstr ""
 
-msgid "Please choose the filament colour"
+msgid "Please choose the filament color"
```

Comment:
Hehe, technically both are correct: one is British English and the other is American English. ;)